solutions/CSES Fixed-Length Paths II.mdx

```diff
@@ -0,0 +1,97 @@
+---
+id: cses-2081
+source: CSES
+title: Fixed-Length Paths II
+author: Andi Qu
+---
+
+**Time Complexity:** $\mathcal O(N \log^2 N)$.
```

Comment:
It's worth noting that you can do this in O(N).

https://cses.fi/paste/871652b7df3be6d6189995/